elements/pfe-clipboard/package-lock.json

```diff
@@ -0,0 +1,304 @@
+{
+  "name": "@patternfly/pfe-clipboard",
```

Comment:
We don't normally commit the package-lock.json files for the individual components.